proto/cache.proto

```diff
@@ -30,13 +36,85 @@ message CacheStats {
   int64 total_cached_action_exec_usec = 11;
 }
 
+// Request to retrieve detailed per-request cache stats.
+message GetCacheScoreCardRequest {
+  context.RequestContext request_context = 1;
+
+  // The invocation ID for which to look up cache stats.
+  string invocation_id = 2;
+
+  // TODO(bduffany): Add paging
```

Comment:
nit: i think we have page token protos -- can you just copy those from elsewhere? should be the same.

Reply:
The only page token proto I could find was in `target_pagination.proto` and the fields will be different. Added a basic page token here